seed/csharp-sdk/alias/src/SeedAlias/SeedAliasClient.cs

```diff
@@ -21,13 +21,23 @@ public SeedAliasClient(ClientOptions? clientOptions = null)
 
     public async Task GetAsync(string typeId)
     {
-        await _client.MakeRequestAsync(
+        var response = await _client.MakeRequestAsync(
             new RawClient.JsonApiRequest
             {
                 BaseUrl = _client.Options.BaseUrl,
                 Method = HttpMethod.Get,
                 Path = $"/{typeId}"
             }
         );
+        if (response.StatusCode is >= 200 and < 400)
```

Comment:
note this change -- we previously weren't even checking response status for endpoints without a response 😲